### PR TITLE
Announce NixOS 22.05

### PR DESCRIPTION
**Pull request by dasJ**
Created as a draft if anyone wants to review this before the release is released

### CODE REVIEW

File: blog/announcements.xml [hunk 1]
```diff
@@ -1,6 +1,93 @@
...
+        like to thank our top 10 contributors:
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/bobby285271">Bobby Rong</a>,
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/jonringer">Jonathan Ringer</a>,
+        <a href="https://github.com/jtojnar">Jan Tojnar</a>,
+        <a href="https://github.com/mic92">Jörg Thalheim</a>,
+        <a href="https://github.com/sternenseemann">sternenseemann</a>,
```
Comment (davidak) [inline blog/announcements.xml:52]:
```suggestion
        <a href="https://github.com/sternenseemann">sternenseemann</a> and
```

File: blog/announcements.xml [hunk 2]
```diff
@@ -1,6 +1,93 @@
...
+        <a href="https://github.com/dotlambda">Robert Schütz</a>.
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
```
Comment (davidak) [inline blog/announcements.xml:60]: it is on place 1 and 3, but not 3

i guess you can just say

```suggestion
        <a href="https://repology.org/repositories/statistics/total">the distribution with the most packages</a>.
```

**Comment by davidak** [comment]
It would be great to have the new graphical installer mentioned, as it is a big change for new users. People might try NixOS out now that haven't before.

You can use my screenshot from https://chaos.social/web/@davidak/108253717600383941 (the CC BY license just requires attribute me (davidak) as the author)
It shown NixOS strength as a desktop distro by providing many desktop options.

this warning has to be removed from the download page as it is no longer the case

Screenshot from 2022-05-30 07-36-54

also:

>The graphical installation ISO image contains NixOS installer as well as Desktop Environment

differentiate between the graphical installer and the manual installation (as i do in https://github.com/NixOS/nixpkgs/pull/175128)

**Comment by davidak** [comment]
There is no official announcement that the graphical installer was added, that's why i would like to have it included in the release announcement.

There where images included before. See the announcements from 2007: https://nixos.org/blog/announcements.html

**Comment by davidak** [comment]
Oh, no. In the screenshot Xfce is selected, but Plasma is shown.

I would also prefer to have the installer window in the right context, so show the complete GNOME desktop (recommended ISO).

The text on the download page also refers to the old installer:

>The graphical installation ISO image contains NixOS installer as

Otherwise it looks good!

**Comment by vlinkz** [comment]
> You can use my screenshot from https://chaos.social/web/@davidak/108253717600383941 (the CC BY license just requires attribute me (davidak) as the author)
> It shown NixOS strength as a desktop distro by providing many desktop options.

I think the version in your screenshot has a slightly older version where the icon in the top bar is messed up. Here's some up-to-date screenshots, you can license them however you see fit.
Screenshot from 2022-05-30 12-03-00
Screenshot from 2022-05-30 12-04-19
Screenshot from 2022-05-30 12-03-22
Screenshot from 2022-05-30 12-04-11
Screenshot from 2022-05-30 12-03-26

**Pull request merged**